cli/sawtooth_cli/network_command/peers.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import os
+import json
+
+from sawtooth_cli.network_command.parent_parsers import base_multinode_parser
+from sawtooth_cli.network_command.parent_parsers import split_comma_append_args
+from sawtooth_cli.network_command.parent_parsers import make_rest_apis
+
+from sawtooth_cli.exceptions import CliException
+
+
+DOT_FILE = 'peers.dot'
+
+
+def add_peers_parser(subparsers, parent_parser):
+    help_text = 'Shows the peering arrangment of a network'
+
+    parser = subparsers.add_parser(
+        'peers',
+        help=help_text,
+        description='{}.'.format(help_text))
+
+    peers_parsers = parser.add_subparsers(
+        title='subcommands',
+        dest='peers_command')
+    peers_parsers.required = True
+
+    _add_list_parser(peers_parsers, parent_parser)
+    _add_graph_parser(peers_parsers, parent_parser)
+
+
+def _add_list_parser(parser, parent_parser):
+    help_text = 'Lists peers for validators with given URLs'
+
+    list_parser = parser.add_parser(
+        'list',
+        help=help_text,
+        description='{}.'.format(help_text),
+        parents=[parent_parser, base_multinode_parser()])
+
+    list_parser.add_argument(
+        'urls',
+        nargs=1,
+        help='the URLs of the validators to be queried',
+        type=str)
```

Comment:
Since you are using `base_multinode_parser()` this creates a duplicate arg:
```
lradmin@frost:~/core-clones/sawtooth-drozd$ ./bin/sawnet peers list --help
usage: sawnet peers list [-h] [-v] [-V] [--users USERNAME[:PASSWORD]]
                         [--pretty]
                         urls [urls ...] urls

Lists peers for validators with given URLs.

positional arguments:
  urls                  The URLs of the validator's REST APIs of interest,
                        separated by commas or spaces. (no default)
  urls                  the URLs of the validators to be queried
```

It looks like you got it right for `peers graph`